src/index.html

```diff
@@ -24,22 +24,36 @@
       </a>
 
       <nav class="navigation">
-        <menu class="navigation__list">
-          <li class="navigation__item"><a href="#apple" class="navigation__link navigation__link--is-active is-active">Apple</a></li>
-          <li class="navigation__item"><a href="#samsung" class="navigation__link">Samsung</a></li>
-          <li class="navigation__item"><a href="#smartphones" class="navigation__link">Smartphones</a></li>
+        <ul class="navigation__list">
```

Comment:
Format your html code with proper indents, line-breaks and spaces.
Checklist #2